pkg/operator/ceph/csi/version.go

```diff
@@ -112,3 +116,29 @@ func extractCephCSIVersion(src string) (*CephCSIVersion, error) {
 
 	return &CephCSIVersion{major, minor, bugfix}, nil
 }
+
+// SupportsCustomCephConf checks if the detected version supports custom ceph.conf
+func (v *CephCSIVersion) SupportsCustomCephConf() bool {
+	// if AllowUnsupported is set also a csi-image greater than the supported ones are allowed
+	if AllowUnsupported {
+		return true
+	}
+
+	if !v.isAtLeast(&cephConfSupportedVersion) {
+		return false
+	}
+
+	if v.Major > cephConfSupportedVersion.Major {
```

Comment:
Doesn't `v.isAtLeast()` on line 127 already cover lines 131-141?

Reply:
Yes this seems like duplicate, Removed it now.